spec.md

```diff
@@ -372,6 +372,8 @@ The Location header will contain the registry URL to access the accepted layer f
 header returns the canonical digest of the uploaded blob which MAY differ from the provided digest. Most clients MAY
 ignore the value but if it is used, the client SHOULD verify the value against the uploaded blob data.
 
+The registry MAY treat the `from` parameter as optional, and it MAY cross-mount the blob if it can be found.
```

Comment:
Should we change `if it can be found` to `if it can be found and can be accessed`?

Reply:
to me, that ACL implication is included in `can be found`, because ideally if it can not be accessed then it should be found. Or is there another use case you're thinking about?

Reply:
Although it is outside of the specification, I would suggest that we indicate that it is not a good idea to implement this feature on any registry with cross-repo ACLs due to the possibility of information disclosure. I do not think `found` implies any notion of "access" in the sense of "Security"

So, I would rather say something like:
"If the registry does not implement a security model which allows for attenuation of access to reading blobs, it MAY treat the `from` parameter as optional, and it MAY cross-mount the blob if it can be found."

Reply:
👍

Reply:
Agreed on adding more to wording here. While we don't define an ACL model, it is a potential pitfall with a very small change that is worth calling out. The updated language is reasonable, but I think it can be simplified some to just mentioned the read access. Like @jonjohnsonjr brought it, it could be that the registry only searches in a list of a blob that are known public and therefore able to be read by anyone.

Reply:
We could leverage this for MCR/ACR as well and for avoiding pushing these public well known layers multiple times.